applications/app/controllers/MediaController.scala

```diff
@@ -52,7 +52,11 @@ class MediaController(contentApiClient: ContentApiClient, val controllerComponen
 
   private def renderMedia(model: MediaPage)(implicit request: RequestHeader): Result = {
     val htmlResponse = () => ContentHtmlPage.html(model)
-    val jsonResponse = () => views.html.fragments.mediaBody(model, displayCaption = false)
```

Comment:
Excuse my ignorance, why does `jsonResponse` contain an HTML fragment?

Reply:
The returned JSON contains an `html` field

Reply:
Can the name be changed to reflect this? Perhaps `htmlFragmentForJsonResponse`?